### PR TITLE
=clu,dc #23340 additional test to see a node take over monitoring of remote DC

### LINKED ISSUE #23340: Additional test for active multi-dc heartbeatsender to activate on other node if oldest goes away
Add a multi-dc test wherein a node that was monitoring the other DC now goes away (leave is fine) and then we check that a different node (e.g. 3rd oldest, if config was 2 oldest nodes monitoring) steps up and becomes active in monitoring

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #23340

Just an additional test, logic worked fine 👍

### CODE REVIEW

File: akka-cluster/src/main/scala/akka/cluster/CrossDcClusterHeartbeat.scala [hunk 1]
```diff
@@ -179,7 +179,7 @@ private[cluster] final class CrossDcHeartbeatSender extends Actor with ActorLogg
   /** Idempotent, become active if this node is n-th oldest and should monitor other nodes */
   private def becomeActiveIfResponsibleForHeartbeat(): Unit = {
     if (!activelyMonitoring && selfIsResponsibleForCrossDcHeartbeat()) {
-      if (verboseHeartbeat) log.debug("Becoming ACTIVE (for DC: {}), monitoring other DCs oldest nodes", selfDataCenter)
+      log.info("Cross DC heartbeat becoming ACTIVE on this node (for DC: {}), monitoring other DCs oldest nodes", selfDataCenter)
```
Comment (johanandren) [inline akka-cluster/src/main/scala/akka/cluster/CrossDcClusterHeartbeat.scala]: Seems useful and reasonable to have at info level 👍

**Review by johanandren — APPROVED** [review]
LGTM, should we have something less of a happy path as well, oldest leaving while there is cross dc unreachability for example?

File: akka-cluster/src/main/scala/akka/cluster/CrossDcClusterHeartbeat.scala (hunk 1 above)
Comment (ktoso) [inline akka-cluster/src/main/scala/akka/cluster/CrossDcClusterHeartbeat.scala]: Right, the becoming active is useful I think, let's keep like this for now 👍

**Comment by ktoso** [comment]
I agree on the additional test too - should we do separate ticket for it or add to https://github.com/akka/akka/issues/23313 ? No preference on my side

**Comment by johanandren** [comment]
Separate ticket is fine

**Comment by ktoso** [comment]
https://github.com/akka/akka/issues/23345 added
Please review @2m if you have some time between your work :)

**Comment by ktoso** [comment]
Merging to avoid PR on top of PR

**Pull request merged**